packages/nextjs/components/scaffold-eth/Contract/ContractVariables.tsx

```diff
@@ -27,14 +27,17 @@ export const ContractVariables = ({
 
   return (
     <>
-      {functionsToDisplay.map(fn => (
-        <DisplayVariable
-          abiFunction={fn}
-          contractAddress={deployedContractData.address}
-          key={fn.name}
-          refreshDisplayVariables={refreshDisplayVariables}
-        />
-      ))}
+      {functionsToDisplay
+        .map(fn => (
+          <DisplayVariable
+            abiFunction={fn}
+            contractAddress={deployedContractData.address}
+            key={fn.name}
+            refreshDisplayVariables={refreshDisplayVariables}
+            inheritedFrom={(deployedContractData.inheritedFunctions as InheritedFunctions)[fn.name]}
+          />
+        ))
+        .sort((a, b) => (b.props.inheritedFrom ? b.props.inheritedFrom.localeCompare(a.props.inheritedFrom) : 1))}
```

Comment:
Maybe we first `.sort`  before `.map` so that : 

First we sort and then create Jsx elements with sorted elements  

In the current situation, we first create JsxElements and then sort them all

Reply:
OK, yes that sounds good.

Reply:
fixed here: https://github.com/FilipHarald/scaffold-eth-2-sandbox/commit/e41ffaaffa0fc4a5a9369d46d3f8224aabf8f731